tox.ini

```diff
@@ -30,6 +30,8 @@ passenv =
 # We removed cleanup from the end because it complicate Travis
 # debugging.
 commands =
+    # molecule for python 2 does not support installing collections
+    ansible-galaxy collection install --requirements-file requirements.yml
```

Comment:
I like this way much better, because we _know why_ it works and it's unlikely to break by third party changes.

Reply:
So telling users to run this will be another change, correct?

Reply:
I think, users will do `ansible-galaxy colllection install pulp_installer` and that should pull the dependencies as described in `galaxy.yml`, right?

Reply:
I guess so, but I never really tested if `ansible-galaxy` would pull the dependencies, I will test it

Reply:
yes, it installs the dependencies:
```
pulp_installer on  ansible210 [$?] via 🐍 v3.8.0 (venv) 
❯ ansible-galaxy collection install pulp-pulp_installer-3.7.0.tar.gz
Starting galaxy collection install process
Process install dependency map
Starting collection install process
Installing 'pulp.pulp_installer:3.7.0' to '/home/faguiar/.ansible/collections/ansible_collections/pulp/pulp_installer'
pulp.pulp_installer (3.7.0) was installed successfully
Installing 'ansible.posix:1.1.1' to '/home/faguiar/.ansible/collections/ansible_collections/ansible/posix'
Downloading https://galaxy.ansible.com/download/ansible-posix-1.1.1.tar.gz to /home/faguiar/.ansible/tmp/ansible-local-229941aoldlpe4/tmpkxkezsjh
ansible.posix (1.1.1) was installed successfully
```

Reply:
Awesome!